ngraph/python/tests/test_onnx/test_ops_reduction.py

```diff
@@ -26,196 +53,17 @@ def import_and_compute(op_type, input_data, **node_attrs):
-
-    assert np.array_equal(import_and_compute("ReduceSum", data, keepdims=0), np.sum(data, keepdims=False))
-
-    assert np.array_equal(
-        import_and_compute("ReduceSum", data, axes=(1,), keepdims=0), np.sum(data, keepdims=False, axis=(1,))
-    )
-
-    assert np.array_equal(
-        import_and_compute("ReduceSum", data, axes=(0, 2), keepdims=0),
-        np.sum(data, keepdims=False, axis=(0, 2)),
-    )
-
-    assert np.array_equal(
-        import_and_compute("ReduceSum", data, axes=(0, 1, 2), keepdims=0),
-        np.sum(data, keepdims=False, axis=(0, 1, 2)),
-    )
-
-
-def test_reduce_sum_keepdims():
-    data = np.array([[[5, 1], [20, 2]], [[30, 1], [40, 2]], [[55, 1], [60, 2]]], dtype=np.float32)
-
-    assert np.array_equal(import_and_compute("ReduceSum", data), np.sum(data, keepdims=True))
-
-    assert np.array_equal(
-        import_and_compute("ReduceSum", data, axes=(1,)), np.sum(data, keepdims=True, axis=(1,))
-    )
-
-    assert np.array_equal(
-        import_and_compute("ReduceSum", data, axes=(0, 2)), np.sum(data, keepdims=True, axis=(0, 2))
-    )
-
-    assert np.array_equal(
-        import_and_compute("ReduceSum", data, axes=(0, 1, 2)), np.sum(data, keepdims=True, axis=(0, 1, 2))
-    )
-
-
-def test_reduce_prod():
-    data = np.array([[[5, 1], [20, 2]], [[30, 1], [40, 2]], [[55, 1], [60, 2]]], dtype=np.float32)
-
-    assert np.array_equal(import_and_compute("ReduceProd", data), np.prod(data, keepdims=True))
-    assert np.array_equal(import_and_compute("ReduceProd", data, keepdims=0), np.prod(data, keepdims=False))
-
-    assert np.array_equal(
-        import_and_compute("ReduceProd", data, axes=(1,)), np.prod(data, keepdims=True, axis=(1,))
-    )
-    assert np.array_equal(
-        import_and_compute("ReduceProd", data, axes=(1,), keepdims=0),
-        np.prod(data, keepdims=False, axis=(1,)),
-    )
-
-    assert np.array_equal(
-        import_and_compute("ReduceProd", data, axes=(0, 2)), np.prod(data, keepdims=True, axis=(0, 2))
-    )
-    assert np.array_equal(
-        import_and_compute("ReduceProd", data, axes=(0, 2), keepdims=0),
-        np.prod(data, keepdims=False, axis=(0, 2)),
-    )
-
-    assert np.array_equal(
-        import_and_compute("ReduceProd", data, axes=(0, 1, 2)), np.prod(data, keepdims=True, axis=(0, 1, 2))
-    )
-    assert np.array_equal(
-        import_and_compute("ReduceProd", data, axes=(0, 1, 2), keepdims=0),
-        np.prod(data, keepdims=False, axis=(0, 1, 2)),
-    )
+@unstrict_xfail_issue_35925
+@pytest.mark.parametrize("operation", reduce_operation_parameters)
+@pytest.mark.parametrize("keepdims", [True, False])
+@pytest.mark.parametrize("axes", reduce_axis_parameters)
+def test_reduce_operation(operation, keepdims, axes):
+    if axes:
+        assert np.array_equal(import_and_compute(operation, reduce_data, axes=axes, keepdims=keepdims),
+                              np.max(reduce_data, keepdims=keepdims, axis=axes))
```

Comment:
You have to also parameterize this numpy reference function.

Reply:
You are absolutely correct, fixed